src/h5web/vis-packs/core/heatmap/ColorBar.module.css

```diff
@@ -1,8 +1,6 @@
 .colorBar {
   display: grid;
   padding: 1rem;
-  color: var(--axis-primary-color);
```

Comment:
This is replaced by the `color` in `.tick text` ?

Reply:
yep